pytorch_lightning/loggers/neptune.py

```diff
@@ -184,7 +189,7 @@ def __init__(
         self.close_after_fit = close_after_fit
         self.experiment_name = experiment_name
         self._kwargs = kwargs
-        self._experiment_id = None
+        self.experiment_id = experiment_id
```

Comment:
experiment_id can't be private here?
```suggestion
        self._experiment_id = experiment_id
```